data/plugins/findfile.lua

```diff
@@ -7,39 +7,64 @@ local config = require "core.config"
 local keymap = require "core.keymap"
 
 config.plugins.findfile = common.merge({
+  -- how many files from the project we store in a list before we stop
   file_limit = 20000,
-  max_search_time = 2.0
+  -- the maximum amount of time we spent gathering files before stopping
```

Comment:
Nit:
```suggestion
  -- the maximum amount of time we spend gathering files before stopping
```

I wonder if we should just let it continue indefinitely. Or maybe this is still useful for weird filesystems (e.g. networked ones)?

Reply:
I'm down to let it continue, but probably not indefinitely. 10s should be enough for anyone. After all, it's meant for quick file searching, I feel like beyond 10s it's not going to be meaningful because most people won't wait that long.

Reply:
Bumped this to 10s.